live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java

```diff
@@ -66,4 +66,30 @@ void testDeleteRequest()
                 }""";
         containerSupport.expectHxIngestMessageReceived(expectedBody);
     }
+
+    @Test
+    void testDeleteRequestWithNoResource()
+    {
+        // given
+        containerSupport.prepareHxInsightToReturnSuccess();
+
+        // when
+        String repoEvent = """
+                {
+                  "specversion": "1.0",
+                  "type": "org.alfresco.event.node.Deleted",
+                  "id": "df329995-d744-427c-bafb-4a31ba7d50e3",
+                  "source": "/08d9b620-48de-4247-8f33-360988d3b19b",
+                  "time": "2021-01-27T10:57:02.586606Z",
+                  "dataschema": "https://api.alfresco.com/schema/event/repo/v1/nodeDeleted",
+                  "datacontenttype": "application/json",
+                  "data": {
+                    "eventGroupId": "acb8e25f-a340-48b5-8de8-249ae5bac670"
```

Comment:
Is it even possible to receive such a Repo event without a resource?

Reply:
No, not really. It's very unlikely anyway.